h/views.py

```diff
@@ -7,10 +7,13 @@
 from pyramid.events import ContextFound
 from pyramid.view import forbidden_view_config, notfound_view_config
 from pyramid.view import view_config
+import pyramid.response
 
 from . import session
 from .models import Annotation
 from .resources import Application, Stream
+import h.api_client.api_client as api_client
```

Comment:
No it's not, I'm an idiot. You need the errors. In which case my opinion stands: it should be `from . import api_client`, or possibly `from .api_client import exceptions as api_exceptions`.